src/jira_client.ts

```diff
@@ -159,11 +171,13 @@ export function JiraAPIClient(
               issuetype: {
                 name: "Task"
               },
-              labels: [`${JIRA_SERVICE_TAG_PREFIX}${serviceId}`].concat(
-                labels || []
-              ),
+              labels: [
+                `${config.service_tag_prefix}${serviceData.serviceId}`,
+                `${config.email_tag_prefix}${serviceData.email}`,
```

Comment:
We can use a custom property to map dev portal account instead of a label. If we use tags these information are shared across all the organization, if we instead use a custom property that info is reachable only to the users that have access to the specific board.

Reply:
Fix with 4de831c

Reply:
@BurnedMarshal @LoFrance it should be searchable/filterable in jira (https://pagopa.atlassian.net/jira/software/projects/IES/boards/118/backlog), is it by custom property?

Reply:
@iwoak it will be searchable with JQL by enabling Ticket tab into the project
<img width="1469" alt="Schermata 2021-08-18 alle 09 35 51" src="https://user-images.githubusercontent.com/3668832/129857890-abd25b7f-db47-407e-8b1b-73d42eb8f797.png">

Reply:
Thanks @BurnedMarshal for the `JQL example`